packages/sampler/stories/default/ViewManager.js

```diff
@@ -0,0 +1,134 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import Group from '@enact/ui/Group';
+import Item from '@enact/ui/Item';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Scroller from '@enact/ui/Scroller';
+import {arrange, SlideArranger, SlideBottomArranger, SlideLeftArranger, SlideRightArranger, SlideTopArranger, ViewManager} from '@enact/ui/ViewManager';
+import {useCallback, useState} from 'react';
+
+import css from './ViewManager.module.less';
+
+const ViewManagerConfig = mergeComponentMetadata('ViewManager', ViewManager);
+
+ViewManagerConfig.defaultProps = {
```

Comment:
Maybe this is not needed...

Reply:
I've added `ViewManagerBase`, but defaultProps prop is not attached. So I think this is needed.

Reply:
As far as I checked, defaultProps is attached well.
But there is no `enteringDelay` default value on the ViewManger component side.
If there is bug about `enteringDelay` let's make new issue for fix this and define default value.
or let's define default value on the ViewManagerBase component.

Reply:
By rechecking it, defaultProps is attached well. I guess there were some mistakes to check. So above code that defines defaultProps is not needed.
Furthermore, `enteringDelay` default value is missing in the component code.
I'd create a new issue to fix it after this is merged.